package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "faas-js-runtime",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

Comment:
no need for this in the PR,  we have some automation that will update the version when we do the release

Reply:
removed version bump